test/katello_test_helper.rb

```diff
@@ -88,7 +88,7 @@ module FixtureTestCase
 
     Setting::Content.load_defaults
 
-    @@admin = ::User.find(FIXTURES['users']['admin']['id'])
+    @@admin = ::User.unscoped.find(FIXTURES['users']['admin']['id'])
```

Comment:
@ares From @johnpmitsch on IRC - "can you add an exception for the class variable so rubocop ignores it?"